purchases/src/main/kotlin/com/revenuecat/purchases/common/subscriberattributes/SpecialSubscriberAttributes.kt

```diff
@@ -37,6 +37,7 @@ internal enum class ReservedSubscriberAttribute(val value: String) {
      */
     MIXPANEL_DISTINCT_ID("\$mixpanelDistinctId"),
     FIREBASE_APP_INSTANCE_ID("\$firebaseAppInstanceId"),
+    TENJIN_ANALYTICS_INSTALLATION_ID("\$tenjinId"),
```

Comment:
Thought about using `$tenjinAnalyticsInstallationId` to be more specific, but the backend is currently expecting this, and I don't think we will need to send anything else anytime soon, so decided to leave it as is.